app/book_display.py

```diff
@@ -0,0 +1,12 @@
+from app.book_model import Book
+
+
+class BookDisplay:
+    @staticmethod
+    def display_book(book: Book, display_type: str) -> None:
+        if display_type == "console":
+            print(book.content)
+        elif display_type == "reverse":
+            print(book.content[::-1])
+        else:
+            raise ValueError(f"Unknown display type: {display_type}")
```

Comment:
Your code violates Single-responsibility principle. You should create separate class for console and reverse dispaly